_data/cryptocurrencies.yml

```diff
@@ -18,7 +18,7 @@ websites:
       img: binance.png
       tfa: Yes
       software: Yes
-      doc: https://support.binance.com/hc/en-us/articles/115002439332-Securing-Your-Trading-Account
+      doc: https://binance.zendesk.com/hc/en-us/articles/115002439332
```

Comment:
Why change domain?
```suggestion
      doc: https://support.binance.com/hc/en-us/articles/115002439332
```

Reply:
Because when I made the semi-automated check, it redirected to biance.zendesk - now it doesn't anymore. So that's fine.